decidim-admin/app/forms/decidim/admin/participatory_process_form.rb

```diff
@@ -29,7 +29,10 @@ class ParticipatoryProcessForm < Form
 
       validate :slug, :slug_uniqueness
 
-      private
+      validates :hero_image, file_size: { less_than_or_equal_to: 10.megabytes }, file_content_type: { allow: ['image/jpeg', 'image/png'] }
+      validates :banner_image, file_size: { less_than_or_equal_to: 10.megabytes }, file_content_type: { allow: ['image/jpeg', 'image/png'] }
```

Comment:
Actually, we probably need to remove the validations from the model and the specs from the model specs